src/sentry/grouping/enhancement-configs/common@2019-03-23.txt

```diff
@@ -56,6 +56,8 @@ family:native package:**/Frameworks/Sentry.framework/**              -app
 # Since 7.0.0, the Cocoa SKD only marks the main executable as inApp. This change doesn't impact
 # applications using static libraries, as when using static libraries, all of them end up in the
 # main executable.
+family:native function:*sentry__hook_dispatch_*                   -app -group
+family:native function:sentry__async_backtrace_capture            -app -group
```

Comment:
Maybe it would be better to just prefix all functions in the PR Cocoa (https://github.com/getsentry/sentry-cocoa/pull/998) with `sentrycrash_` then we wouldn't need to add these rules and we could ditch this PR as we already have a rule for `sentrycrash_`?

Reply:
:-D that is a good idea actually